src/coll_score/ucc_coll_score.c

```diff
@@ -414,7 +414,7 @@ static ucc_status_t str_to_msgranges(const char *str, size_t **ranges,
 
 static ucc_status_t ucc_coll_score_parse_str(const char       *str,
                                              ucc_coll_score_t *score,
-                                             ucc_rank_t        team_size)
+                                             ucc_rank_t        team_size)//NOLINT
```

Comment:
@vspetrov Adding NOLINT directive to the code seems like distraction? Is there a different way that is less distractive to add the directives?

Reply:
I dont know another way. Those are "unused vars" in the functions that are not fully implemented. We either need to suppress these false positives, OR remove check for unused vars from Linter. @manjugv @Sergei-Lebedev thoughts?

Reply:
I think NOLINT here is  just another reminder that something is not implemented, so it looks fine to me